docs/whats-new/zowe-v3-migration.md

```diff
@@ -0,0 +1,172 @@
+
+### Migrating from Zowe v2.9.0 or Lower
+
+If you are migrating from Zowe **v2.9.0** or a lower version, it is recommended to delete the `<zowe.workspaceDirectory>/app-server/plugins` directory so that it can be regenerated on the next run of Zowe.
+In this version and prior there were old and no longer used Application Framework plugins and references to them will complicate logs with harmless errors.
+
+### Migrating from Zowe v2.3.0 or Lower
+
+If you are running Zowe **v2.3.0** or a lower version, a **clean install** of Zowe v3 is highly recommended to avoid potential issues during the migration process.
+
+
+## V3 Prerequisite Changes
+
+Before starting the migration, ensure the following system requirements are met:
+
+- **z/OSMF**  
+Version V2R5 with APAR PH12143 or V3R1 is required. JWT support for z/OSMF must be enabled, [as seen in their documentation.](https://www.ibm.com/docs/en/zos/3.1.0?topic=configurations-enabling-json-web-token-support)
+- **Java**  
+Java 17 is required. The Zowe YAML parameter `java.home` value should be a **Java 17** home location. If an administrator uses `zwe init` to set up Zowe, ensure the `java` for that user is v17 by including it in the `PATH` environment variable.
+- **Node.js**  
+The Zowe YAML parameter `node.home` value should be a **Node.js 18 or 20** home location, as Node 16 and below are no longer supported.
+
+
+## System and Security Changes
+
+- Existing SAF settings for Zowe do not need to be changed for v3, so install steps such as `zwe init security`, the job or workflow ZWESECUR, and the jobs ZWEIRAC, ZWEITSS, and ZWEIACF are not required to be re-run.
+
+- Existing keyrings and keystores do not need to be changed for v3, so install steps such as `zwe init certificate`, the job or workflow ZWEKRING or jobs starting with ZWEIKR* are not required to be re-run.
+
+- The following network changes are needed for added or removed servers:
+
+| Component name | Change | Default Port | Default Jobname | Details |
+|----------------|--------|--------------|-----------------|---------|
+| zaas | Added | 7558 | ZWE1AZ | This component is responsible for authentication and is now required when using the API Mediation Layer |
+| metrics-service | Removed | 7551 | ZWE1MS | This service has been deprecated and removed. Currently, no replacement is available. The Open Telemetry standard will be implemented later, which will serve as a replacement |
+| jobs-api | Removed | 7558 | ZWE1EJ | This component was deprecated in Zowe v2 and is now removed. Ensure that you switch to using equivalent z/OSMF endpoints |
+| files-api | Removed | 7559 | ZWE1EF | This component was deprecated in Zowe v2 and is now removed. Ensure that you switch to using equivalent z/OSMF endpoints |
+| cloud-gateway | Removed | 7563 | ZWE1CG | The cloud-gateway has replaced the main gateway component in v3 and is now accessible on 7554 as ZWE1AG |
```

Comment:
this detail about cloud-gateway replaces gateway yet takes on the same name and port as gateway is perhaps too much technical information for users - I shared this document with users and they didn't know what it meant.

Reply:
Then we should just mention that it is removed here

Reply:
Reverted it to the previous phrase "The cloud-gateway has been removed as a standalone component and merged into the gateway", looks ok?